### ISSUE TITLE
ReferenceError: window is not defined

### ISSUE THREAD

**Issue by irudoy** [zerobias/telegram-mtproto#95]
```
ReferenceError: window is not defined
    at exports.tsNow (/.../node_modules/telegram-mtproto/lib/service/time-manager.js:27:30)
    at generateMessageID (/.../node_modules/telegram-mtproto/lib/service/time-manager.js:32:19)
    at new NetMessage (/.../node_modules/telegram-mtproto/lib/service/networker/net-message.js:35:47)
    at NetworkerThread.wrapApiCall (/.../node_modules/telegram-mtproto/lib/service/networker/index.js:232:19)
    at MapAction._monadT.MaybeT.toFuture.chain.map.networker [as mapper] (/.../node_modules/telegram-mtproto/lib/service/invoke.js:45:176)
    at MapAction.resolved (/.../node_modules/telegram-mtproto/node_modules/fluture/index.js:684:82)
    at resolved (/.../node_modules/telegram-mtproto/node_modules/fluture/index.js:175:19)
    at Resolved._fork (/.../node_modules/telegram-mtproto/node_modules/fluture/index.js:572:3)
    at drain (/.../node_modules/telegram-mtproto/node_modules/fluture/index.js:232:25)
    at settle (/.../node_modules/telegram-mtproto/node_modules/fluture/index.js:161:17)
    at Sequence.interpreter [as _fork] (/.../node_modules/telegram-mtproto/node_modules/fluture/index.js:246:3)
    at /.../node_modules/telegram-mtproto/node_modules/fluture/index.js:382:58
    at Promise (<anonymous>)
    at Sequence.Future$promise [as promise] (/.../node_modules/telegram-mtproto/node_modules/fluture/index.js:382:10)
    at action.thru.map.map.map.filter.filter.tap.map.i (/.../node_modules/telegram-mtproto/lib/state/epic/net-request.js:94:58)
    at MapSink.event (/.../node_modules/telegram-mtproto/node_modules/most/lib/fusion/Map.js:71:22)
    at TapSink.event (/.../node_modules/telegram-mtproto/node_modules/most/lib/combinator/transform.js:81:13)
    at FilterSink.event (/.../node_modules/telegram-mtproto/node_modules/most/lib/fusion/Filter.js:51:21)
    at MapSink.event (/.../node_modules/telegram-mtproto/node_modules/most/lib/fusion/Map.js:71:13)
    at FilterMapSink.event (/.../node_modules/telegram-mtproto/node_modules/most/lib/fusion/FilterMap.js:35:21)
    at tryEvent (/.../node_modules/@most/multicast/dist/multicast.js:24:10)
    at MulticastSource.event (/.../node_modules/@most/multicast/dist/multicast.js:85:5)
    at PropagateTask.emit [as _run] (/.../node_modules/most/lib/scheduler/PropagateTask.js:58:8)
    at PropagateTask.run (/.../node_modules/most/lib/scheduler/PropagateTask.js:43:8)
    at ScheduledTask.run (/.../node_modules/most/lib/scheduler/ScheduledTask.js:20:20)
    at runTask (/.../node_modules/most/lib/task.js:18:17)
    at runTasks (/.../node_modules/most/lib/scheduler/Timeline.js:80:7)
    at Timeline.runTasks (/.../node_modules/most/lib/scheduler/Timeline.js:69:20)
    at Scheduler._runReadyTasks (/.../node_modules/most/lib/scheduler/Scheduler.js:107:17)
    at Asap.Scheduler._runReadyTasksBound [as f] (/.../node_modules/most/lib/scheduler/Scheduler.js:29:10)
    at Asap.run (/.../node_modules/most/lib/scheduler/ClockTimer.js:32:30)
```

**Comment by zerobias** [comment]
Sorry, will fix soon!

**Comment by zerobias** [comment]
Fixed in `3.2.10`
